src/filters/FilterNumbers.jsx

```diff
@@ -67,95 +75,109 @@ export default class FilterNumbers extends React.Component {
+      // const value = e.target.value < min ? min : e.target.value;
+
+      this.props.dispatch(filterChanged(this.props.fieldName, {userMax: e.target.value}));
+    };
   }
   updateSlider(values) {
     this.props.dispatch(filterChanged(this.props.fieldName, {userMin: values[0], userMax: values[1]}));
   }
-  renderGTLT() {
+  renderHelpText() {
+    let help = "";
 
+    if (this.props.userMax < this.props.userMin) {
+      help = "Max cannot be less than min"
+    }
+
+    return help;
+  }
+  render() {
     const clampedUserMin = clamp(this.props.userMin, this.props.min, this.props.max);
     const clampedUserMax = clamp(this.props.userMax, this.props.min, this.props.max);
 
-    var min = (this.props.isPercentage) ? Math.floor(clampedUserMin * 100) : clampedUserMin;
-    var max = (this.props.isPercentage) ? Math.floor(clampedUserMax * 100) : clampedUserMax;
+    const min = clampedUserMin;
+    const max = clampedUserMax;
 
     return (
-      <div>
-        <span
-          style={{
-            float: 'left',
-            position: 'relative',
-            top: -6
-          }}>
-          {min} - {max}{this.props.isPercentage ? '%' : ''}
-        </span>
-        <div style={{marginTop: 12, marginLeft: 70, width: 200}}>
-          <Slider
-            min={this.props.min}
-            max={this.props.max}
-            step={this.state.step}
-            defaultValue={[clampedUserMin, clampedUserMax]}
-            value={[clampedUserMin, clampedUserMax]}
-            onChange={this.updateSlider.bind(this)}
-            withBars/>
-        </div>
-      </div>
-    );
-  }
-  renderEQUALS() {
-    return (
-      <div>
-        {/* will be a component */}
-        <span> {this.props.description} </span>
-        <span style={style.symbol} onClick={this.handleSymbolClick.bind(this)}>{'='}</span>
-        <input
-          onFocus={this.popEqualsSlider}
-          style={style.sliderInput}
-          onChange={this.handleEqualChanged.bind(this)}
-          value={this.state.equals || this.props.min}/>
-      </div>
-    );
-  }
-  render() {
-    return (
       <Card>
-        <CardHeader>{this.props.description}</CardHeader>
+        <div style={{
+          marginTop: 0,
+          marginBottom: 10,
+          display: "flex",
+          alignItems: "flex-start",
+          justifyContent: "space-between"
+        }}>
+        <span style={{marginBottom: 10, marginRight: 20}}>{this.props.description}</span>
         {
-          this.state.symbol === 'GTLT' ?
-            this.renderGTLT() :
-            this.renderEQUALS()
+          this.props.distributions ?
+          <Sparkline
+            distribution={
+              this.props.distributions[this.props[this.props.fieldName].field]
+            }/> :
+            ""
         }
+        </div>
+        <div>
+          <input
+            onChange={this.onMinChanged().bind(this)}
```

Comment:
I'm gonna leave this one. It's equiv, and if we want to do fancy things with clamp / calculations before we pass the events through, we need it